game/scripts/vscripts/components/boss/ngp.lua

```diff
@@ -17,50 +22,115 @@ function NGP:Init ()
   })
 
   CustomGameEventManager:RegisterListener('ngp_selection', Dynamic_Wrap(NGP, 'PlayerVote'))
+
+
+  for playerId = 0,19 do
+    local player = PlayerResource:GetPlayer(playerId)
+    if player ~= nil then
+      if player:GetTeam() == 3 then
+        totalbadplayers = totalbadplayers + 1
+      elseif player:GetTeam() == 2 then
+        totalgoodplayers = totalgoodplayers + 1
```

Comment:
Getting the total number of players on a team can be done using the built-in `PlayerResource:GetPlayerCountForTeam` function.